frontend/src/components/topbar.tsx

```diff
@@ -66,6 +76,7 @@ export const Topbar = () => {
         <div className="flex justify-end items-center gap-2">
           <MobileDropdown />
           <OmniSearch setOpen={setOmniOpen} className="lg:hidden" />
+          <Docs/>
           <Version />
```

Comment:
The styling between Docs and Version has too big a gap, it doesn't look quite right. This is because those Docs and Version Button each have some `padding x`, and the container div has `gap-2`, combined they are too much space.

Wrapping <Docs /> and <Version /> in another div with `gap-0` works:

```
<div className="flex gap-0">
  <Docs />
  <Version />
</div>
```